ui/webui/test/prepare-updates-img

```diff
@@ -6,8 +6,8 @@ set -eu
 # This will change once we include this changes upstream and start building boot.iso with the new dependencies
 # Then we can safely remove this workaround
 COCKPIT_URLS='
-    https://dl.fedoraproject.org/pub/fedora/linux/development/38/Everything/x86_64/os/Packages/c/cockpit-ws-287-1.fc38.x86_64.rpm
-    https://dl.fedoraproject.org/pub/fedora/linux/development/38/Everything/x86_64/os/Packages/c/cockpit-bridge-287-1.fc38.x86_64.rpm
+    https://kojipkgs.fedoraproject.org//packages/cockpit/287/1.fc39/x86_64/cockpit-ws-287-1.fc39.x86_64.rpm
```

Comment:
I think it makes more sense to install the f38 cockpit in the fedora-38 branch

Reply:
Yes, woops!